js/ui/status/automaticUpdates.js

```diff
@@ -280,7 +280,7 @@ var Indicator = new Lang.Class({
 
         // Create the notification
         this._notification = new MessageTray.Notification(source, title, subtitle, { gicon: gicon });
-        this._notification.setUrgency(MessageTray.Urgency.NORMAL);
+        this._notification.setUrgency(alreadySentNotification ? MessageTray.Urgency.NORMAL : MessageTray.Urgency.CRITICAL);
```

Comment:
I'd like to see a comment explaining our intent here.

Reply:
Oops -- your comment came in a minute after I merged.  I'll push a follow-up commit with a comment.

Reply:
Here's the updated comment:
```
        // Create the notification.
        // The first time we notify the user for a given connection,
        // we set the urgency to critical so that we make sure the
        // user understands how we may be changing their settings.
        // On subsequent notifications for the given connection,
        // for instance if the user regularly switches between
        // metered and unmetered connections, we set the urgency
        // to normal so as not to be too obtrusive.
```